doc/source/user_guide/plotting_with_bokeh.md

```diff
@@ -0,0 +1,135 @@
+  language: python
+  name: python3
+---
+
+(plotting_with_bokeh)=
+
+# Plotting with Bokeh
+
+Arviz has the option to use {doc}`Bokeh <bokeh:index>` as backend which offers interactive plots. Although most of the functions in the {ref}`plot <plot_api>` module work seamlessly with any backend, some advanced plots may require the use of backend specific features. In this guide, advanced plotting with Bokeh will be covered.
+
+This page can be downloaded as a {jupyter-download:script}`Python script <plotting_with_bokeh>`
+or as a {jupyter-download:nb}`Jupyter notebook <plotting_with_bokeh>`.
+
+```{code-cell} ipython3
+import arviz as az
+import matplotlib.pyplot as plt
+import numpy as np
+
+az.style.use("arviz-darkgrid")
+
+# Confgure Bokeh as backend
+az.rcParams["plot.backend"] = "bokeh"
+az.output_notebook()
+```
+
+:::{note}
+To display the plots as output in a Jupyter notebook, it's required to call the function  {func}`arviz.output_notebook` beforehand.
+:::
+
+:::{warning} Page in construction
+:::
+
+## Customizing plots
+
+
+### Using `backend_kwargs`
+
+The `backend_kwargs` argument can be very useful for some specific configuration. That is parameters available in {class}`bokeh:bokeh.plotting.figure.Figure`.
+
+As an example, the following code changes the color background and the plot width.
+
+```{code-cell} ipython3
+# load data
+data = az.load_arviz_data('radon')
+```
+
+```{code-cell} ipython3
+az.plot_posterior(
+    data,
+    var_names=["g"],
+    backend_kwargs={"width": 350, 
+                    "background_fill_color": "#d3d0e3"});
+```
+
+### The parameter `show`
+
+The parameter `show` is used to control whether the plot is displayed or not. This behavior can be useful when a plot will be displayed in a grid as the example in the following section.
+
+
+### Defining custom axes
+
+The `ax` argument of any `plot` function allows to use created axes manually. In the example, this parameter allows to arrange 2 different plots in a grid, set limit to the x axis and share the axes' ranges between the two plots.
+
+
+```{code-cell} ipython3
+from bokeh.io import show
+from bokeh.layouts import gridplot, row
+from bokeh.plotting import figure
+
+# load data
+observed_data = data.observed_data.y.to_numpy()
+# create axes
+f1 = figure(x_range=(observed_data.min() - 1, observed_data.max() + 1))
+f2 = figure(x_range=f1.x_range, y_range=f1.y_range)
+# plot
+az.plot_ppc(data, group="prior", num_pp_samples=100, show=False, ax=f1)
+az.plot_ppc(data, group="posterior", num_pp_samples=100, show=False, ax=f2)
+
+show(gridplot([[f1], [f2]]))
+```
+
+## Combining ArviZ and Bokeh plots
+
+Arviz plot returns a {class}`~bokeh.plotting.Figure` object, therefore different Bokeh plots can be combined into a plot created using Arviz:
+
+```{code-cell} ipython3
+# load data
+data = az.load_arviz_data('regression1d')
+X = data.observed_data.y_dim_0.to_numpy()
+Y = data.observed_data.y.to_numpy()
+predictions = data.posterior_predictive.y.to_numpy()
```

Comment:
```suggestion
y_pp = data.posterior_predictive.y.to_numpy()
```

or `post_pred`, `pps`, `y_post_pred`, inverted order... The comment is basically about avoiding the use of the word "predictions" in that context as it can be confusing. In ArviZ we have a group called `posterior_predictive` and a group called `predictions`. Both are used to store posterior predictive samples that can also/always be called predictions if you want, but we make this distinction to reduce the complexity of arviz code. 

The posterior_predictive group should store pp samples aiming to predict the same data that was already observed (kind of -but not really- as if it hadn't been observed) which might seem a bit pointless but is very useful for model criticism. These samples are therefore used to perform posterior predictive checks: comparing the actual observed data to these samples because if the model is correct, it should not be possible to distinguish these synthetic/simulated observations from the real ones.

The predictions group stored pp samples that aim to predict on new/unobserved data

Reply:
Variable renamed

Reply:
And now I got the names...a bit confusing 😅